recipes/pyRingGW/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "pyRingGW" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyRingGW-{{ version }}.tar.gz
+  sha256: f354b2ee1104d09091c1ea1ca7f1d05ad08aa8f0fd1c09bc0a8605917a6b5ad6
+
+build:
+  skip: true  # [py<36]
```

```suggestion
  skip: true  # [py<36]
  skip: true  # [win]
```
Please skip Windows if it is not possible to build on Windows.

---

Addressed [here](https://github.com/conda-forge/staged-recipes/pull/21577/commits/1ba3cf2305479c8cfb8b6c4dfff1d0263884cd1d).